### PR TITLE
feat(ui): open message list to unread position 

### PR DESCRIPTION
**Pull request by imtoori**

### CODE REVIEW

File: packages/stream_chat_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
   color of pinned messages.
 - Added unread messages divider in `StreamMessageListView`.
 - Added `StreamMessageListView.unreadMessagesSeparatorBuilder`.
-
+- Now `StreamMessageListView` opens to the oldest unread message by default.
```
Comment (ayush221b) [inline packages/stream_chat_flutter/CHANGELOG.md:19]: Maybe leave an empty line after this

File: packages/stream_chat_flutter/lib/src/message_list_view.dart [hunk 2]
```diff
@@ -1395,6 +1407,7 @@ class _StreamMessageListViewState extends State<StreamMessageListView> {
             );
           });
         } else if (streamChannel?.channel.state?.unreadCount != 0) {
+          print('increasing unread');
```
Comment (ayush221b) [inline packages/stream_chat_flutter/lib/src/message_list_view.dart:1410]: Please remove the print call

**Review by ayush221b — CHANGES_REQUESTED** [review]
Minor changes requested

File: packages/stream_chat_flutter/lib/src/message_list_view.dart (hunk 2 above)
Comment (imtoori) [reply packages/stream_chat_flutter/lib/src/message_list_view.dart:1410]:
```suggestion
```

File: packages/stream_chat_flutter/CHANGELOG.md (hunk 1 above)
Comment (imtoori) [reply packages/stream_chat_flutter/CHANGELOG.md:19]:
```suggestion
- Now `StreamMessageListView` opens to the oldest unread message by default.

```

**Review by ayush221b — APPROVED** [review]

**Pull request merged**